apps/metric_vnode/eqc/metric_vnode_eqc.erl

```diff
@@ -181,23 +178,23 @@ prop_handoff() ->
                    os:cmd("mkdir data"),
                    {S, T} = eval(D),
 
-                   List = ?T:to_list(T),
+                   List = gb_trees:to_list(T),
 
                    List1 = [{get(S, Time, 1), V} || {Time, V} <- List],
                    List2 = [{unlist(mmath_bin:to_list(Vs)), Vt} || {{_ ,Vs}, Vt} <- List1],
                    List3 = [true || {_V, _V} <- List2],
 
                    Fun = fun(K, V, A) ->
-                                 [?V:encode_handoff_item(K, V) | A]
+                                 [metric_vnode:encode_handoff_item(K, V) | A]
                          end,
                    FR = ?FOLD_REQ{foldfun=Fun, acc0=[]},
                    {async, {fold, AsyncH, _}, _, S1} =
-                       ?V:handle_handoff_command(FR, self(), S),
+                       metric_vnode:handle_handoff_command(FR, self(), S),
                    L = AsyncH(),
                    L1 = lists:sort(L),
-                   {ok, C, _} = ?V:init([1]),
+                   {ok, C, _} = metric_vnode:init([1]),
                    C1 = lists:foldl(fun(Data, SAcc) ->
-                                            {reply, ok, SAcc1} = ?V:handle_handoff_data(Data, SAcc),
+                                            {reply, ok, SAcc1} = metric_vnode:handle_handoff_data(Data, SAcc),
```

Comment:
Line 197 is too long:                                             {reply, ok, SAcc1} = metric_vnode:handle_handoff_data(Data, SAcc),.